lib/tokens.js

```diff
@@ -449,6 +465,32 @@ export async function resolveToInternalId({token}) {
   return {internalId};
 }
 
+/**
+ * Updates a token batch based off of `internalId`.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.internalId - The internal ID of the token batch.
+ * @param {number} options.minAssuranceForResolution - Minimum level of
+ *   assurance required for token resolution.
+ *
+ * @returns {boolean} Returns true if update occurred.
+ */
+export async function updateBatch({internalId, minAssuranceForResolution}) {
```

Comment:
```suggestion
export async function updateBatch({internalId, minAssuranceForResolution} = {}) {
```

Also, can't there be multiple batches open for `internalId`?

Reply:
Is it ok for closed batches to be updated?

Reply:
Updated to update multiple batches.

What exactly do you mean by "closed batches"? I have added a check for `remainingTokenCount: {$gt:0}` and the batches will automatically be deleted when they expire, is there another definition for `closed`?